src/Vegetation/BigLeaf/bigleaf_parameterizations.jl

```diff
@@ -0,0 +1,271 @@
+                            a2::FT,
+                            Rd::FT,
+                            β::FT) where {FT}
+    An = maximum([0, minimum([a1*β, a2]) - Rd])
+    return An
+end
+
+"""
+dark_respiration(Vcmax::FT,
+                 β::FT) where {FT}
+
+Computes dark respiration (`Rd`) as a function of the maximum rate of carboxylation of Rubisco (`Vcmax`),
+and the moisture stress factor (`β`). 
+"""
+function dark_respiration(Vcmax::FT,
+                          β::FT) where {FT}
+    Rd = 0.015*Vcmax*β
+    return Rd
+end
+
+"""
+    GPP(An::FT,
+        K::FT,
+        LAI::FT,
+        Ω::FT) where {FT}
+
+Computes the total canopy photosynthesis (`GPP`) as a function of 
+the total net carbon assimilation (`An`), the extinction coefficient (`K`),
+leaf area index (`LAI`) and the clumping index (`Ω`).
+"""
+function GPP(An::FT,
+             K::FT,
+             LAI::FT,
+             Ω::FT) where {FT}
+    GPP = An*(1-exp(-K*LAI*Ω))/K
+    return GPP
+end
+
+"""
+    MM_Kc(Kc25::FT,
+          ΔHkc::FT,
+          T::FT,
+          To::FT,
+          R::FT) where {FT}
+
+Computes the Michaelis-Menten coefficient for co2 (`Kc`)
+as a function of its value at 25 °C (`Kc25`),
+a constant (`ΔHkc`), a standard temperature (`To`),
+the unversal gas constant (`R`), and the temperature (`T`).
+"""
+function MM_Kc(Kc25::FT,
+               ΔHkc::FT,
+               T::FT,
+               To::FT,
+               R::FT) where {FT}
+    Kc = Kc25*exp(ΔHkc*(T-To)/(To*R*T))
+    return Kc
+end
+
+"""
+    MM_Ko(Ko25::FT,
+          ΔHko::FT,
+          T::FT,
+          To::FT,
+          R::FT) where {FT}
+
+Computes the Michaelis-Menten coefficient for O2 (`Ko`)
+as a function of its value at 25 °C (`Ko25`),
+a constant (`ΔHko`), a standard temperature (`To`),
+the universal gas constant (`R`), and the temperature (`T`).
+"""
+function MM_Ko(Ko25::FT,
+         ΔHko::FT,
+         T::FT,
+         To::FT,
+         R::FT) where {FT}
+    Ko = Ko25*exp(ΔHko*(T-To)/(To*R*T))
+    return Ko
+end
+
+"""
+    fVcmax(Vcmax25::FT,
+           Ta::FT,
+           To::FT,
+           R::FT) where {FT}
+
+Computes the maximum rate of carboxylation of Rubisco (`Vcmax`) 
+as a function of air temperature (`Ta`), Vcmax at the reference temperature 25 °C (`Vcmax25`),
+the universal gas constant (`R`), and the reference temperature (`To`).
+"""
+function fVcmax(Vcmax25::FT,
+                Ta::FT,
+                To::FT,
+                R::FT) where {FT}
+    Vcmax = Vcmax25*exp(50*(Ta-To))/(R*Ta)
+    return Vcmax
+end
+
+# 3. Stomatal conductance model
+
+"""
+    Monin_Obukhov(ρa::FT,
+                  geff::FT,
+                  qs::FT,
+                  qv::FT) where {FT}
+
+Computes canopy transpiration (`Trans`) following the Monin-Obukhov theory, 
+i.e., as a function of the specific humidity (`qa`)
+the saturated specific humidity over liquid water (`qv`),
+and the effective conductivity (`geff`).
+""" 
+function Monin_Obukhov(ρa::FT,
```

Comment:
We actually wont need this formula - it is evaluated in SurfaceFluxes.jl, we can just call it there.

Reply:
so we remove this function?